spec.html

```diff
@@ -39284,7 +39200,7 @@ <h1>Promise Resolve Functions</h1>
             1. Let _thenAction_ be _then_.[[Value]].
             1. If IsCallable(_thenAction_) is *false*, then
               1. Return FulfillPromise(_promise_, _resolution_).
-            1. Perform EnqueueJob(*"PromiseJobs"*, PromiseResolveThenableJob, &laquo; _promise_, _resolution_, _thenAction_ &raquo;).
+            1. Perform EnqueueJob(HostEnqueuePromiseJob, PromiseResolveThenableJob, &laquo; _promise_, _resolution_, _thenAction_ &raquo;).
```

Comment:
Maybe `HostEnqueuePromiseJob(JobCreate(...))` would be a bit clearer?

Reply:
I'm trying to preserve the general layout that the previous specification had, as recommended in the TC39 meeting. I think it'd be possible to do further editorial changes based on these sorts of ideas as a follow-on PR.

Reply:
JobCreate is just EnqueueJob renamed, is that too much churn?

Reply:
I'd like to leave that possible further revision for a separate patch. In this version, I'm trying to work towards the conclusion that @erights and I reached with TC39. It's not like what you're proposing is incorrect; this is just full of different value judgements about the editorial form, and I'd like to get something landed so that we can fix the original layering issue.